Scripts/InitEnvironment.swift

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/swift
+import Foundation
+
+let currentPath = "./"
+
+func writeCodeInFile(filePath: String, codes: String) {
+    let fileURL = URL(filePath: filePath)
+    guard let writeHandle = try? FileHandle(forWritingTo: fileURL) else {
+        fatalError("\(filePath)을 찾을 수 없습니다")
+    }
+    writeHandle.seek(toFileOffset: 0)
+    try? writeHandle.write(contentsOf: Data(codes.utf8))
+    try? writeHandle.close()
+}
+
+func envString(projectName: String, organizationName: String) -> String {
+    return """
+import ProjectDescription
+
+public struct ProjectEnvironment {
+    public let name: String
+    public let organizationName: String
+    public let deploymentTarget: DeploymentTarget
+    public let platform: Platform
+    public let baseSetting: SettingsDictionary
+}
+
+public let env = ProjectEnvironment(
+    name: "\(projectName)",
+    organizationName: "\(organizationName)",
+    deploymentTarget: .iOS(targetVersion: "16.0", devices: [.iphone, .ipad]),
```

Comment:
```suggestion
    deploymentTarget: .iOS(targetVersion: "15.0", devices: [.iphone, .ipad]),
```
여기 부분은 15버전으로 안 바꿔도 되는건가요?

Reply:
InitEnv는 안바꿨네 ㄹㅇㅋㅋ